### PR TITLE
Cleaned up some dead code, reduced default to 128 megs, added Megs unit.

### PR DESCRIPTION
**Pull request by phoglund**
Addressing Phillip's comment on the last PR.

### CODE REVIEW

File: src/content/datachannel/datatransfer/index.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
     <section>
       <div>
-        <input type="number" id="megsToSend" min="1" name="megs" value="1048"/>
+        <input type="number" id="megsToSend" min="1" name="megs" value="128"/> Megs
```
Comment (samdutton) [inline src/content/datachannel/datatransfer/index.html]: I think this should be MB, and should be in a label element. (Can't remember why the progress labels are in divs – had a good reason which I've now forgotten...)

Also (nitpick) I think the button should probably be left aligned above the input and progress element, to match demos like [this](http://webrtc.github.io/samples/src/content/datachannel/basic/). 

So:

    <div id="button">
        <button>...    
    </div>
    <div class="label">Data size: </div>
    <input ...>
    <label for="megsToSend">megabytes</label>

(The input/label content will be displayed on a single line.)

In css/main.css: 

    div#button {
        margin: 0 0 1em 0;
    }

(You can also get rid of the form selector in that file.)

File: src/content/datachannel/datatransfer/index.html [hunk 2]
```diff
@@ -46,7 +46,7 @@
...
         <button id="sendTheData" type="button">Generate and send data</button>
```
Comment (juberti) [inline src/content/datachannel/datatransfer/index.html]: Suggest adding a checkbox for ordered/unordered. We don't have any unordered demos, and we had a crasher in this area recently. The rest of the demo should work fine as-is in either case.

File: src/content/datachannel/datatransfer/index.html (hunk 1 above)
Comment (phoglund) [inline src/content/datachannel/datatransfer/index.html]: Done.

File: src/content/datachannel/datatransfer/index.html (hunk 2 above)
Comment (phoglund) [inline src/content/datachannel/datatransfer/index.html]: Ok, sure. How do I know it actually worked to go into unordered mode? I'm passing {ordered: true/false} into createDataChannel now based on the value of the checkbox.

**Comment by phoglund** [comment]
PTAL

**Comment by samdutton** [comment]
LGTM except that I get an error when the transfer completes:

    Uncaught InvalidStateError: Failed to execute 'send' on 'RTCDataChannel':
    RTCDataChannel.readyState is not 'open'main.js:83 generateData

There's also a minor code formatting glitch: there should be curly brackets for the if statement on main.js line 41.

**Comment by phoglund** [comment]
Fixed nits and the invalid state error.

**Comment by samdutton** [comment]
LGTM.

**Pull request merged**